plugins/scaffolder-backend/package.json

```diff
@@ -81,12 +81,13 @@
     "isomorphic-git": "^1.23.0",
     "jsonschema": "^1.2.6",
     "knex": "^2.0.0",
+    "libsodium-wrappers": "^0.7.11",
     "lodash": "^4.17.21",
     "luxon": "^3.0.0",
     "morgan": "^1.10.0",
     "node-fetch": "^2.6.7",
     "nunjucks": "^3.2.3",
-    "octokit": "^2.0.0",
+    "octokit": "^2.0.14",
```

Comment:
revert?

Reply:
i could have sworn this one was required but let me revert and give it another go.  Too many plates spinning at the same time (and some unlucky timing)